powergenome/load_construction.py

```diff
@@ -0,0 +1,673 @@
+pop_cols = ["ipm_region", "state", "state_prop"]
+# pop_dtypes = {"State Prop": np.float32}
+pop = pd.read_parquet(
+    path_in / "ipm_state_pop_weight_20210517.parquet", columns=pop_cols
+)
+# pop = pop.astype(pop_dtypes)
+states = pop.drop_duplicates(subset=["state"])["state"]
+states_abb = list(map(state2abbr, states))
+pop["state"] = list(map(state2abbr, pop["state"]))
+states_eastern_abbr = [
+    "ME",
+    "VT",
+    "NH",
+    "MA",
+    "RI",
+    "CT",
+    "NY",
+    "PA",
+    "NJ",
+    "DE",
+    "MD",
+    "DC",
+    "MI",
+    "IN",
+    "OH",
+    "KY",
+    "WV",
+    "VA",
+    "NC",
+    "SC",
+    "GA",
+    "FL",
+]
+states_central_abbr = [
+    "IL",
+    "MO",
+    "TN",
+    "AL",
+    "MS",
+    "WI",
+    "AR",
+    "LA",
+    "TX",
+    "OK",
+    "KS",
+    "NE",
+    "SD",
+    "ND",
+    "IA",
+    "MN",
+]
+states_mountain_abbr = ["MT", "WY", "CO", "NM", "AZ", "UT", "ID"]
+states_pacific_abbr = ["CA", "NV", "OR", "WA"]
+states_eastern = list(map(abbr2state, states_eastern_abbr))
+states_central = list(map(abbr2state, states_central_abbr))
+states_mountain = list(map(abbr2state, states_mountain_abbr))
+states_pacific = list(map(abbr2state, states_pacific_abbr))
+
+# some parameters
+stated_states = ["New Jersey", "New York", "Virginia"]
+# Date Jan 29, 2021
+# (2) PA, NJ, VA, NY, MI all have EV and heat pump stocks from NZA DD case
+# consistent with their economywide decarbonization goals.
+# https://www.c2es.org/content/state-climate-policy/
+# Date Feb 10, 2021
+# Remove high electrification growth in PA and MI in stated policies;
+# they dont have clean energy goals so kind of confusing/inconsistent to require high electrification in these states.
+# So our new "Stated Policies" definition for electrification is states
+# with BOTH economywide emissions goals + 100% carbon-free electricity standards
+# = NY, NJ, VA.
+stated_states_abbr = list(map(state2abbr, stated_states))
+# years = ["2022", "2025", "2030", "2040", "2050"]
+cases = ["current_policy", "stated_policy", "deep_decarbonization"]
+
+running_sector = [
+    "Residential",
+    "Residential",
+    "Commercial",
+    "Commercial",
+    "Transportation",
+    "Transportation",
+    "Transportation",
+    "Transportation",
+]
+running_subsector = [
+    "space heating and cooling",
+    "water heating",
+    "space heating and cooling",
+    "water heating",
+    "light-duty vehicles",
+    "medium-duty trucks",
+    "heavy-duty trucks",
+    "transit buses",
+]
+Nsubsector = len(running_subsector)
+
+logger = logging.getLogger(__name__)
+
+
+def CreateOutputFolder(case_folder):
+    path = case_folder / "extra_outputs"
+    if not os.path.exists(path):
+        os.makedirs(path)
+
+
+######################################
+# CreatingBaseLoad.R
+def CreateBaseLoad(
+    years: List[int],
+    regions: List[str],
+    output_folder: Path,
+    path_growthrate: Path = None,
+) -> None:
+    path_processed = path_in
```

Comment:
Change this to a single year and IPM region so that the results are easier to reuse?